source/features/reactions-avatars.tsx

```diff
@@ -58,18 +69,24 @@ const viewportObserver = new IntersectionObserver(changes => {
 });
 
 function showAvatarsOn(commentReactions: Element): void {
-	const reactionTypes = $$('.social-reaction-summary-item', commentReactions).length;
+	const reactionTypes = $$([
+		'.social-reaction-summary-item',
+		'[class^="Tooltip__TooltipBase"]',	// React views, isIssue
+	], commentReactions).length;
 	const avatarLimit = arbitraryAvatarLimit - (reactionTypes * approximateHeaderLength);
 
 	const participantByReaction
-		= $$(':scope > button.social-reaction-summary-item', commentReactions)
+		= $$([
+			':scope > button.social-reaction-summary-item:enabled',
```

Comment:
`:enabled` broke it on locked issues:

- https://github.com/refined-github/refined-github/issues/8081